tweetapi/login.py

```diff
@@ -78,6 +81,9 @@ def get(self,request):
         return response
 
 class ChangePassword(CreateAPIView,TwitterUserPermission):
```

Comment:
I thought you said you took out the permission inheritance? I won't merge this PR until you take it out

Reply:
i did sir, they where pointed in the views i think i didnt look at the change password too, sorry about it sir

Reply:
fixed sir